examples/simpleapp/src/main/resources/sc_settings.conf

```diff
@@ -55,6 +55,10 @@ scorex {
 
     forger {
         automaticForging = false
+        closedForge = false
+        forgerList = [
```

Comment:
here, I believe `forgerList` expected to be empty.

Reply:
Done